public/doc/swagger-2.yaml

```diff
@@ -531,7 +531,7 @@ parameters:
     name: portfolio_id
     in: path
     description: The Portfolio ID
-    required: false
+    required: true
```

Comment:
@mkanoor - So the problem with `required: false` is based on the fact that the `in` attribute references `path`?

Reply:
@syncrou All path parameters are required, Query parameters are optional.